src/commands/install.yml

```diff
@@ -33,7 +33,7 @@ steps:
             name: Install SFDX - Standalone
             command: |
               cd /tmp
-              wget https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz
+              wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
               mkdir sfdx
               tar xJf sfdx-linux-amd64.tar.xz -C sfdx --strip-components 1
               ./sfdx/install
```

Comment:
there will be problems here, the new version doesn't have install file

Reply:
updated to set path env var via $BASH_ENV